src/toolchain/web-guide.md

```diff
@@ -0,0 +1,147 @@
+<!--
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+
+# Intro to Web Builds
+
+Web builds are a fair bit more difficult to get started with compared to native builds.
+This will be a complete guide on how to get things compiled.
+
+Information to consider before starting:
+
+- Setting up a web server to host and share your game is considered out of scope of this guide, and is best explained elsewhere.
+- Web support with `gdext` is experimental and should be understood as such before proceeding.
+
+---
+
+
+## Installing Pre-Requisites
+
+- Install a nightly build of `rustc`, the `wasm32-unknown-emscripten` target for `rustc`, and `rust-src`.
+Assuming that Rust was installed with `rustup`, this is quite simple.
+
+  ```sh
+  rustup toolchain install nightly
+  rustup component add rust-src --toolchain nightly
```

Comment:
See here: https://discord.com/channels/723850269347283004/1134478713002655876/1176743545768521738

Reply:
It wasn't necessary for me, but someone in the discord somehow didn't have rust-src.  So, I think best to leave it there as a just in case? Shouldn't do anything if it's already installed, at least.

Reply:
Yea I was installing everything from scratch in a Linux VM and rustup didn't seem to install it by default.

Reply:
You're right, [it looks like it's required](https://rust-lang.github.io/rustup/concepts/components.html#components):

> `rust-src` — This is a local copy of the source code of the Rust standard library. This can be used by some tools, such as [RLS](https://github.com/rust-lang/rls), to provide auto-completion for functions within the standard library; [Miri](https://github.com/rust-lang/miri/) which is a Rust interpreter; **and Cargo's experimental [build-std](https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#build-std) feature**, which allows you to rebuild the standard library locally.